cmd/owl/main.go

```diff
@@ -204,12 +215,20 @@ func main() {
 					// Write is running only once
 					if ev.Op == fsnotify.Chmod {
 
+						// check if is set filter
+						if viper.GetString("filter") != "" {
+							rgxp := regexp.MustCompile(viper.GetString("filter"))
```

Comment:
Compile the `filter` string only once (at the start of the program when we recognize that filter flag was set.